shielddash/core/static/css/site.styl

```diff
@@ -143,14 +156,23 @@ p {
   &.study-titles > *:not(.study-field-faded) {
     color: $normal-text;
   }
+  b {
+    font-weight: 400;
+
+    &.percent {
+      color: $green-text;
+      display: none;
+    }
+  }
 }
 
 .study-channel-totals-row {
   border-top: 2px solid rgba(0,0,0,0.1);
 }
 
-.study-field-faded {
-  color: rgba(0,0,0,0.4);
+.study-field-faded,
+.study-field-faded .percent {
+  color: rgba(0,0,0,0.4) !important;
```

Comment:
What does the `!important` override here? Would it be possible to override with specificity instead. If nothing else, maybe we could document the reason for `!important`.